lib/decimal.ml

```diff
@@ -311,6 +311,15 @@ let is_finite = function Finite _ -> true | _ -> false
 let is_infinite = function Finite _ -> false | _ -> true
 let is_signed = function Finite { sign = Neg; _ } | Inf Neg -> true | _ -> false
 
+let is_integral = function
+  | Finite { exp; _ } when exp >= 0 -> true
+  | Finite { coef; exp; _ } ->
+    (* [exp] is negative so we negate it to index into [coef] *)
+    let idx = Int.neg exp in
```

Comment:
It looks like the `Int` module was included starting with version 4.08, which is also the minimum bound in the OPAM file for this package. I can change this, though, in case that's ever lowered.